docs/conf/prague/2019/news/cfp-diversity-tickets.rst

```diff
@@ -0,0 +1,47 @@
+:template: {{year}}/generic.html
+
+.. post:: Apr 25, 2019
+   :tags: prague-2019, tickets, cfp, diversity
+
+Call for Proposals Closing Soon, Diversity Tickets Available
+============================================================
+
+Greetings, fellow documentarians in Europe and beyond! In this post we're happy to announce our diversity scholarship initiative and remind you that the call for proposals is about to close, so if you want to submit your proposal now is the time.
+
+Tickets are selling and we're happy to see so many of our documentarian friends on the attendee list already! Check out the `Tickets <https://www.writethedocs.org/conf/prague/2019/tickets/>`_ page for pricing and other details. Last year we sold out at 280 attendees, and due to venue capacity we are only able to increase our available tickets to 300, to help our attendees feel comfortable in this beautiful venue.
+
+Submit your talk by 30th April
+------------------------------
+
+Thank you so much to all who submitted their proposals so far! Our Call for Proposals is open until **Midnight CET, 30th April 2019**. Make sure to get your talk in before the deadline!
+
+The `Call for Proposals <https://www.writethedocs.org/conf/prague/2019/cfp/>`_ page has tips and examples to help inspire you.
+And if you have an idea but aren't sure what to do with it, feel free to ask for feedback on `Slack <http://slack.writethedocs.org/>`_. Your fellow documentarians will be happy to help!
+
+Speaking at Write the Docs is a fantastic way to share your ideas with our inspiring community of documentarians.
+If there is something you’d really like to see a talk on this year, submit a proposal on it, or refer someone else who would be good!
```

Comment:
I don't really like "someone else who would be good" for some reason.
How about:
> .. or refer someone else.
or:
> .. or refer someone else who would be a good speaker.

No strong feelings.